src/components/pages/HomePage.jsx

```diff
@@ -1,28 +1,115 @@
-import { useNavigate } from 'react-router-dom';
+import React from 'react';
+import { useNavigate } from "react-router-dom";
+import mainImage from "../../assets/imgMainPage.jpg";
+import GreenBlobImage from "../../assets/blob_green.png";
+import YellowBlobImage from "../../assets/blob_yellow.png";
+import PurpleBlobImage from "../../assets/blob_dusty_purple.png";
+import items from "../../data/data"
+import ArrowUp from "../../assets/icons/icons-arrow-up.png";
+import ArrowSearch from "../../assets/icons/icon-search.svg";
 
-const HomePage = () => {
+const scrollToTop = () => {
+  window.scrollTo({
+    top: 0,
+    behavior: 'smooth',
+  });
+};
 
+const HomePage = () => {
   const navigate = useNavigate();
+
   const search = () => {
     navigate("/search");
   };
-  const login = () => {
-    navigate("/login");
-  };
-  const register = () => {
-    navigate("/register");
-  };
-
 
   return (
-    <>
-      <h1 className="text-3xl font-spartan font-bold">HOME PAGE</h1>
-      <button onClick={search}>Search classes</button>
-      <button onClick={login}>Log In</button>
-      <button className='bg-red hover:bg-pureWhite hover:text-red hover:border-2 hover:border-red text-white font-spartan font-semibold text-lg py-1 px-7 rounded-lg' 
-              onClick={register}>Join</button>
-    </>
+    <div className="flex flex-col my-auto lg:mx-auto my-12">
+      <section className="flex flex-col lg:flex-row lg:justify-between xl:justify-between justify-center items-center">
+        <div className="lg:text-left text-center items-center justify-center xl:justify-between">
+          <h1 className="text-4xl sm:text-5xl md:text-6xl lg:text-4xl xl:text-5xl items-center font-spartan font-semibold tracking-wide leading-tight pt-3 mb-8">
+            Helping unlock
+            <br />
+            your talent with
+            <br />
+            the best teachers
+          </h1>
+          <div className="flex justify-center lg:justify-start">
+          <button
+            className="flex items-center bg-red hover:bg-pureWhite hover:text-red hover:border-2 hover:border-red text-white font-spartan font-semibold text-2xl py-4 px-8 rounded-lg"
```

Comment:
To make button more responsive, you can remove px-8, change py-4 to py-2, add width w-3/5 and justify center